templates/webapi/NewJob/create.html.ep

```diff
@@ -0,0 +1,122 @@
+% layout 'bootstrap';
```

Comment:
We don't use camel-cased path elements here. So please move this file under `test`. (Is it really that hard to follow the established conventions of a projects. Please don't do everything your way. If everyone would do that this project would end up as a total mess of inconsistencies. **And it becomes really annoying having to state this so often.**)

Reply:
Now I noticed. believe me if I say I didnt try to do anything on my way here. rather trying to find my way. and what happened briefly I create this somewhere else and then I copied and remained as such because I didnt noticed